internal/services/containers/kubernetes_cluster_node_pool_resource_test.go

```diff
@@ -1060,6 +1060,21 @@ func TestAccKubernetesClusterNodePool_gpuInstance(t *testing.T) {
 	})
 }
 
+func TestAccKubernetesClusterNodePool_gpuProfile(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_kubernetes_cluster_node_pool", "test")
+	r := KubernetesClusterNodePoolResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.gpuProfile(data),
```

Comment:
Matching up the test name to the new property:
```suggestion
			Config: r.gpuDriver(data),
```